doc/concepts.qbk

```diff
@@ -199,6 +199,76 @@ the generator, for example to re-run a test suite at a later time.]
 
 [endsect]
 
+[section Quasi-Random Number Generator]
+
+A quasi-random number generator is a __UniformRandomNumberGenerator which
+provides a deterministic sequence of quasi-random numbers, based on some
+algorithm and internal state. [classref boost::random::niederreiter_base2_engine
+Niederreiter base 2] generator is an example of such a [qrng quasi-random
+number generator]. The "quasi" modifier is used to denote more clearly that the
+values produced by such a generator are neither random nor pseudo-random, but
+they form a low discrepancy sequence. The intuitive idea is that a low discrepancy
+sequence is more evenly distributed than a pseudo random sequence would be.
+For example, if we generate a low discrepancy sequence of 2D points on a square,
+this square would be covered more evenly, and the number of points falling to any
+part of the square would be proportional to the number of points in the whole square.
+Such sequences share some properties of  random variables and in certain applications
+such as the quasi-Monte Carlo method their lower discrepancy is an important advantage.
+
+[note Quasi-Monte Carlo method uses a low-discrepancy sequence such as the
+Niederreiter base 2 sequence, the Sobol sequence, or the Faure sequence among the others.
+The advantage of using low-discrepancy sequences is a probabilistically faster rate of
+convergence. Quasi-Monte Carlo has a rate of convergence close to '''<code>O(1/N)</code>''',
+whereas the rate of convergence for the Monte Carlo method, which uses a pseudo-random sequence,
+is '''<code>O(N<superscript>-0.5</superscript>)</code>'''.]
+
```

Comment:
Quasi-Monte Carlo integration converges roughly as '''<code>O(1/N)</code>''', whereas the convergence of naive Monte Carlo integration, which uses pseudo-random sequences, is only '''<code>O(N<superscript>-0.5</superscript>)</code>'''.]

Reply:
No. The suggestion is less precise. Consider what "converges roughly as O(1/N)" is supposed to mean?

Reply:
Maybe return to the correct estimate of O(log(N)^d/N), as the logarithmic factors dominate for achievable N and somewhat large d.